nservicebus/tools/index.md

```diff
@@ -5,3 +5,7 @@ reviewed: 2020-05-29
 ---
 
 - [Migrating timeouts](migrate-to-native-delivery.md)
+
+## Support policy
+
+Only the latest version of a given tool is supported and users should consider upgrading before each use.
```

Comment:
```suggestion
Since tools are meant to be run on an as-needed basis, only the latest version of a given tool is supported and users should consider upgrading before each use.
```